src/lib/autoinstall/autosetup_helpers.rb

```diff
@@ -151,6 +144,7 @@ def autosetup_network
       end
 
       Yast::WFM.CallFunction("lan_auto", ["Write"]) if network_before_proposal?
+      @network_configured = true
```

Comment:
What if the `Write` operation fails? Should we set the configured flag only after success? 

Maybe something like
```ruby
@network_configured = Yast::WFM.CallFunction("lan_auto", ["Write"]) if network_before_proposal?
```
would do the trick... (but maybe it would be better to split it, it's quite complicated and hard to read...)

Reply:
In that case, i guess trying to autosetup the network later will produce the same write error , so, it is just a flag indicating that autosetup_network was already called, somehow like the self_update that is only run once. So, it is like a oneshot run independently of the result.